Adds new guns

AtomicWorm opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->
After many years of research, humanity has finally re-discovered the lost technology of double-stack pistol magazines!

This PR does a few things:
- Adds double-stack 9mm and .45 pistol magazines, 15 and 12 rounds, respectively. They cannot be used by guns that use single stack magazines;
- Adds a new gun for antagonists: 9mm combat pistol. It uses double-stack magazines and costs 40 telecrystals. Each magazine costs 6 telecrystals;
- Repurposes the currently unused officer sidearm .45 into a pistol using double-stack magazines. The sidearm crate from Supply now comes with these, and the ammunition crates come with magazines for them;
- Lowers the ammunition capacity of single stack 9mm magazines to 8 rounds, and lowers the telecrystal cost from 4 to 3;
- Raises the telecrystal cost of the 10mm SMG magazine to 8, so it's now in line with other SMG magazines;
- Adds the 9mm combat pistol to random traders and spawns, as well as possible spawn weapons for raiders and renegades. Basically, everywhere the holdout pistol could show up, the combat pistol can now as well;
- Adds the new magazine types to the hacked autolathe list.

What the new pistols look like, with their magazines:
![temp](https://user-images.githubusercontent.com/13301593/41578392-6b016c0c-7368-11e8-9a0e-e41b3b53f5eb.png)

AtomicWorm committed:
Adds double-stack pistol magazines and guns that use them

comma commented:
>The sounds effects use are from the game "Insurgency", and therefore ©New World Interactive.
I claim no ownership whatsoever over the sound effects, and do not intend to profit from their use in any way at all.

Nope can't use, incompatible with our assets stuff, gotta be public domain or CC

AtomicWorm commented:
Removed.

While this sat here I decided to update the .45 pistol's sprite so it looks more like what it was meant to be: an USP, the blockiest pistol that isn't a Glock. Also added the magazines to the image.

babydoll commented on maps/torch/datums/supplypacks/security.dm:
```diff
@@ -55,6 +55,15 @@
 	containername = "\improper NanoTrasen light armor crate"
 	access = access_nanotrasen
 
+/decl/hierarchy/supply_pack/security/pistol
```
Why'd you move this here, out of interest?

Also, since these guns have twice the capacity as before, their price (and their ammo's) should probably be increased accordingly.

AtomicWorm replied:
I moved it there because both the Torch-specific and general crates would show up on the list otherwise. As for the increase in price, I believe the pistols themselves are fine, as the next gun in the scale of lethality is 20 points higher, and they come loaded with flash rounds, which are entirely useless, but I agree about the ammunition, so I'll raise the cost to 30 points rather than 20.
